src/static/routes-to-urls/__tests__/index.js

```diff
@@ -81,24 +79,24 @@ test("routes to urls", (t) => {
       "/key/value2",
       "/one",
       "/two",
-    ]
-  )
-})
+    ])
+  })
 
-const routesNoMatches = (
-  <Route component={ Noop }>
-    <Route path="/no-match/:lol" component={ Noop } />
-    <Route path="*" component={ Noop } />
-  </Route>
-)
+  it("log message if there is no matches", () => {
+    jest.resetModules()
+    const routesNoMatches = (
+      <Route component={ Noop }>
+        <Route path="/no-match/:lol" component={ Noop } />
+        <Route path="*" component={ Noop } />
+      </Route>
+    )
 
-test("routes to urls without matches", (t) => {
-  t.plan(1)
-  routesToUrls(routesNoMatches, collection, {
-    log: (message) => {
-      t.truthy(message.includes(
-        "It looks like some parameters can't be mapped to create routes:  :lol"
-      ))
-    },
+    const log = jest.fn()
+    routesToUrls(routesNoMatches, collection, { log })
+
+    expect(log).toBeCalled()
+    expect(log.mock.calls[0][0]).toContain(
```

Comment:
Here for example, we were passing a callback directly. No need to use jest mock since you can pass a log function. I am just not sure how to handle the t.plan pattern.